site/src/assets/data/faq/index.js

```diff
@@ -1,14 +1,70 @@
+/* eslint-disable no-multi-str */
 const data = {
     faqs: [
+
+// Copy and paste the following template 
+// for new Q/A entries 
+/*
+        {
+            question: "",
+            category: "Meshery Playground",
+            answer: [
+                "",
+                "",
+            ],
+        },
+*/
+
+        {
+            question: "Why do I need to sign-up/register to use the Cloud Native Playground?",
+            category: "Meshery Playground",
+            answer: [
+                "Sign-up to receive a user account, entitled with perpetually free access, because the Playground is not a crypto mine. The Meshery Playground is connected to live Kubernetes cluster(s) and allows users full-control over those clusters. ",
```

Comment:
TLDR, let's make point straight away.
This account will give you free and perpetual access to the Playground. The reason for signing up is that the Playground is not a cryptocurrency mining platform. It is actually connected to live Kubernetes clusters, and by signing up, you gain full control over those clusters.

Reply:
@AugustasV, you're very much right about this suggestion. The other phrasing is indirect and leaves much to the imagination of the reader.

Reply:
Rephrased.